src/ixdat/techniques/ms.py

```diff
@@ -500,12 +528,126 @@ def gas_flux_calibration_curve(
             n_dot_fit = np.array([0, max(n_dot_vec)])
             S_fit = n_dot_fit * pfit[0] + pfit[1]
             ax.plot(n_dot_fit * 1e9, S_fit * 1e9, "--", color=color)
-        cal = CalPoint(mol=mol, mass=mass, F_type=cal_type, F=F, date=self.yyMdd)
+        cal = MSCalResult(
+            name=f"{mol}@{mass}",
+            mol=mol,
+            mass=mass,
+            cal_type=cal_type,
+            F=F,
+        )
         if return_ax:
             return cal, ax
         return cal
 
-    def multicomp_gas_flux_calibration(
+    def siq_gas_flux_calibration(self, mol, mass, tspan, chip=None):
+        """Simple pure-gas flux calibration, using external MS quantification package
+
+        Args:
+            mol (str): Name of molecule to be calibrated (e.g. "He")
+            mass (str): Mass at which to calibrate it (e.g. "M4")
+            tspan (timespan): A timespan during which the pure gas is in the chip
+            chip (Chip, optional): An object defining the capillary inlet, if different
+                than the standard chip assumed by the external package.
+
+        Returns CalPoint: An object from the external MS quantification package,
+           representing the calibration result
+        """
+        if not plugins.use_siq:
+            raise QuantificationError(
+                "`MSMeasurement.gas_flux_calibration_siq` only works when using an "
+                "external MS quantification package "
+                "(`ixdat.options.activate_siq()`). "
+                "For native ixdat MS quantification, `gas_flux_calibration` has to be"
```

Comment:
This is outdated (for native ixdat MS quantification, the function has to be called without the siq)

Reply:
fixed.